INTEGRATION [PR#425 > development/1.1] Improvement/doc zenko 1224

bert-e opened the pull request:
This pull request has been created automatically.
It is linked to its parent pull request #425.

**Do not edit this pull request directly.**
If you need to amend/cancel the changeset on branch
`w/1.1/improvement/doc-ZENKO-1224`, please follow this
procedure:

```bash
 $ git fetch
 $ git checkout w/1.1/improvement/doc-ZENKO-1224
 $ # <amend or cancel the changeset by _adding_ new commits>
 $ git push origin w/1.1/improvement/doc-ZENKO-1224
```

Please always comment pull request #425 instead of this one.

wabernat committed:
Added Zenko logo. Got rid of unneeded css-junk.

Adding this line to satisfy the linter.

wabernat committed:
Added Zenko logo. Got rid of unneeded css-junk.

wabernat committed:
Merge branch 'improvement/doc-ZENKO-1293-add-logos' of https://github.com/scality/Zenko into improvement/doc-ZENKO-1293-add-logos

wabernat committed:
Added versioned object deletion script description.

Also found, corrected a bad link (link and reference).

bert-e committed:
Merge branch 'improvement/doc-ZENKO-1224' into tmp/octopus/w/1.1/improvement/doc-ZENKO-1224

ghost commented:
There were the following issues with this Pull Request

* Commit: 37c94e57cac34c148cf07c1b4e994aba2747dd76
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 12f330b96fe0e4ed780a6ca9fafb1eefe41c5490
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: ded2619a31e5f7550baa1fd1ef5540c3ae5ee70a
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

wabernat committed:
improvement: ZENKO-1224: add s3utils instructions for deleting versioned files

Also found a bad link. To fix in one file, had to fix (add an anchor) in a second.

wabernat committed:
Merge branch 'improvement/doc-ZENKO-1224' of https://github.com/scality/Zenko into improvement/doc-ZENKO-1224

bert-e committed:
Merge branch 'improvement/doc-ZENKO-1224' into tmp/octopus/w/1.1/improvement/doc-ZENKO-1224

ghost commented:
There were the following issues with this Pull Request

* Commit: 37c94e57cac34c148cf07c1b4e994aba2747dd76
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: 12f330b96fe0e4ed780a6ca9fafb1eefe41c5490
  - ✖ message may not be empty
  - ✖ type may not be empty
* Commit: ded2619a31e5f7550baa1fd1ef5540c3ae5ee70a
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues
